learning_journal/learning_journal/scripts/initializedb.py

```diff
@@ -0,0 +1,50 @@
+import os
+import sys
+import transaction
+from datetime import datetime
+from .entries import ENTRIES
+
+from pyramid.paster import (
+    get_appsettings,
+    setup_logging,
+    )
+
+from pyramid.scripts.common import parse_vars
+
+from ..models.meta import Base
+from ..models import (
+    get_engine,
+    get_session_factory,
+    get_tm_session,
+    )
+from ..models import MyEntry
+
+
+def usage(argv):
+    cmd = os.path.basename(argv[0])
+    print('usage: %s <config_uri> [var=value]\n'
+          '(example: "%s development.ini")' % (cmd, cmd))
+    sys.exit(1)
+
+
+def main(argv=sys.argv):
+    if len(argv) < 2:
+        usage(argv)
+    config_uri = argv[1]
+    options = parse_vars(argv[2:])
+    setup_logging(config_uri)
+    settings = get_appsettings(config_uri, options=options)
+
+    engine = get_engine(settings)
+    Base.metadata.drop_all(engine)  # credit Justin Lange
+    Base.metadata.create_all(engine)
+
+    session_factory = get_session_factory(engine)
+
+    with transaction.manager:
+        dbsession = get_tm_session(session_factory, transaction.manager)
+
+        for entry in ENTRIES:
+
+            entry = MyEntry(title=entry["title"], body=entry["body"], creation_date=datetime.strptime(entry["creation_date"], '%b %d, %Y'))
```

Comment:
very good.  This is exactly right.  I would consider putting the args to the `MyEntry` constructor on different lines for readability:

```python
entry = MyEntry(
    title=entry["title"],
    body=entry["body"],
    creation_date=datetime.strptime(entry["creation_date"], '%b %d, %Y')
)
```